src/__experimental__/components/input/input-presentation.spec.js

```diff
@@ -169,7 +174,7 @@ describe('InputPresentation', () => {
       assertStyleMatch({
         background: '#d9e0e4',
         borderColor: '#d9e0e4 !important'
-      }, renderWithTheme({ disabled: true }, classicTheme));
+      }, renderWithTheme({ disabled: true }, classicTheme).find(InputPresentationStyle));
```

Comment:
Same with the theme here.

Reply:
If changing this is trivial then go ahead but if it would involve significant changes to style assertions or chromatic snapshots I'd rather go with this as-is and change it as part of the classic deprecation please.

Reply:
@DipperTheDan To be hones I'd leave that until the `classicTheme` is removed from the `input-presentation` component

Reply:
That's fine and totally understand. Just figured I'd ask the question. Shy bairns get nothing 😉